modules/lvms-about-adding-devices-to-a-vg.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-adding-devices-to-a-vg_{context}"]
+= About adding devices to a volume group
+
+The `deviceSelector` field in the `LVMCluster` CR contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.
+
+You can specify the device paths in the `deviceSelector.paths` field, the `deviceSelector.optionalPaths` field, or both. If you do not specify the device paths in both the `deviceSelector.paths` field and the `deviceSelector.optionalPaths` field, {lvms} adds the supported unused devices to the LVM volume group. 
+
+The devices that you want to add to the volume group must be supported by {lvms}. For information about unsupported devices, see _Devices not supported by {lvms}_ in _Additional resources_.
+
+If you do not add the `deviceSelector` field in the `LVMCluster` CR, {lvms} automatically adds the new devices when the devices are available.
+
+{lvms} adds the devices to the LVM volume group only if the following conditions are met:
+
+* The device path exists.
+* The device is supported by {lvms}.
+
+You can also add the path to the RAID arrays to integrate the RAID arrays with {lvms}. For more information, see _Integrating RAID arrays with {lvms}_ in _Additional resources_. 
```

Comment:
I believe the citations throughout this PR need to be switched to " " and not italics. See [ISG](https://www.ibm.com/docs/en/ibm-style?topic=references-citations#:~:text=%22Requirements%22%20on%20page,9%2C%20%22Programming%20techniques%22.)

Reply:
Updated as per the suggestion.